From ae1b199ce4763bf5dc7225ecbfa82dad2857cc77 Mon Sep 17 00:00:00 2001
From: Kosta Zertsekel <konszert@marvell.com>
Date: Wed, 14 Aug 2013 10:44:37 +0300
Subject: [PATCH 0952/1825] fix: neta: amc: change source for number fo TCONTs
 of PON port

https://github.com/MISL-EBU-System-SW/misl-windriver.git linux-3.4.69-14t2-read
commit 22b0756ab4fd64bd335d1307c34009ec600c4987

	- Pass number of TCONTs for PON port as HalInit structure
	- Don't use CONFIG_MV_PON_TCONTS define

Change-Id: I3ca832ff4e6059c9829e0d1641cd9cc238847b73
Signed-off-by: Kosta Zertsekel <konszert@marvell.com>
Signed-off-by: Zhong Hongbo <hongbo.zhong@windriver.com>
---
 arch/arm/plat-feroceon/mv_hal/neta/gbe/mvNeta.h |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/arch/arm/plat-feroceon/mv_hal/neta/gbe/mvNeta.h b/arch/arm/plat-feroceon/mv_hal/neta/gbe/mvNeta.h
index 5a9ac64..0c3ee7a 100644
--- a/arch/arm/plat-feroceon/mv_hal/neta/gbe/mvNeta.h
+++ b/arch/arm/plat-feroceon/mv_hal/neta/gbe/mvNeta.h
@@ -319,6 +319,10 @@ typedef struct {
 extern MV_NETA_PORT_CTRL **mvNetaPortCtrl;
 extern MV_NETA_HAL_DATA mvNetaHalData;
 
+#ifndef CONFIG_MV_PON_TCONTS
+# define CONFIG_MV_PON_TCONTS 1
+#endif
+
 #ifdef CONFIG_MV_PON
 #define MV_ETH_MAX_TCONT() 	CONFIG_MV_PON_TCONTS
 #endif /* CONFIG_MV_PON */
-- 
1.7.5.4

